### ISSUE TITLE
How 'input.txt' should look like? 

### ISSUE THREAD

**Issue by daridar** [epfml/sent2vec#38]
Hello! I found your answer about how each sentence should look like: 

> it is important for machine_learning 

But if I want to build my own model, what should I feed to sent2vec? 

`../sent2vec/fasttext sent2vec -input data/documents_mixed.txt -output my_model -minCount 8 -dim 700 -epoch 10 -lr 0.2 -wordNgrams 2 -loss ns -neg 10 -thread 36 -t 0.000005 -dropoutK 4 -minCountLabel 20 -bucket 4000000`

How does each sentence need to be separated from each other?

For example: 

> 1 sentence: "i love my dog" 
> 2 sentence: "my dog loves me" 
> 
> input.txt: "i love my dog", "my dog loves me"  
> or: "i love my dog" "my dog loves me"  
> or "i love my dog"  (new line) 
>      "my dog loves me"  

Thank you in advance! 

And one more question:
For a very large corpus (84 Gb - tweets and web docs) what are your recommendations for parameters t, dropoutK and bucket? 
And is it possible to change linearly learning rate to particular min learning rate value?

**Comment by martinjaggi** [comment]
yes, training file needs to be one sentence per line, see https://github.com/epfml/sent2vec#training-new-models

the corpus size you mention should be fine we'd assume (maybe try to tune the hyper-parameters on a smaller subset of it, and start from our default ones)

hope this helps a bit

**Comment by martinjaggi** [comment]
yes

**Comment by daridar** [comment]
Thank you!
